src/bundle.js

```diff
@@ -0,0 +1,46 @@
+import Mfs from 'memory-fs'
+import webpack from 'webpack'
+import { fromCallback } from 'bluebird'
+import { resolve, join, relative } from 'path'
+
+const
+  isObject = (x) => typeof x === 'object',
+  isString = (x) => typeof x === 'string'
+
+export async function bundle (compiler, next) {
```

Comment:
I think I'll do that. Will just need to make a note that if the userland patch is authored in commonjs that they use `module.exports.default` for the patch